openedx/features/survey_report/management/commands/generate_report.py

```diff
@@ -0,0 +1,28 @@
+"""
+CLI command to generate survey report.
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+
+from openedx.features.survey_report.api import generate_report
+
+
+class Command(BaseCommand):
+    """
+    Management command to generate a new survey report with
+    non-sensitive data.
+    """
+
+    help = """
+        This command will create a new survey report using some
+        models to get the total number oof courses offered, currently active learners,
+        learners ever registered, and generated certificates.
+        """
+
+    def handle(self, *_args, **_options):
+        try:
+            generate_report()
+        except Exception as error:
+            raise CommandError(f'An error has occurred while report was generating. {error}') from error
```

Comment:
not to confuse with course grade reports.
Could you change it to `.. survey report was generating...`

Reply:
@felipemontoya this was changed, thanks for your comments.